rancher/manifest.yaml

```diff
@@ -0,0 +1,5 @@
+name: Rancher
+version: stable
```

Comment:
We always prefer to lock this down to a particular version - so if Rancher (in this case) breaks something upstream at least we have a known good version here.